### ISSUE TITLE
Can you please explain to me how to run this?

### ISSUE THREAD

**Issue by Zer0v87** [programa-stic/snapchat-decrypt#8]
Hi there, I admire your work and greatly appreciate it. However I am no noob, but I am also not that experienced in doing things like this. Can you please explain to me step by step on how to run this. Thank you as that would be greatly appreciate. From what seeing this looks great and I want to run this script.

**Comment by jheguia** [comment]
Hi,
  If you're using Snapchat 9.1.2.0 or later, the script will not work anymore. See previous issue.

Cheers,
   Juan

**Comment by Zer0v87** [comment]
I haven't tried the  script yet as I don't know what to do to.  However I did root my phone, downloaded python on my PC and installed the framework on my phone. My Snapchat is also below version 8.

**Comment by jheguia** [comment]
You just run the script, as in 
$ python decrypt_photos.py
Keep in mind you should have the version of ADB that runs as root (see previous issues)

**Comment by Zer0v87** [comment]
So I have I have to run the script on my PC, with my phone plugged into it. Right?

**Comment by jheguia** [comment]
Yes. Maybe you should first check that the ADB program is the right one.

**Comment by Zer0v87** [comment]
Alrights thanks for all the help! I appreciate it. And yes my ADB program is up to date.  (Version 2 insecure)
